src/dune_engine/build_system.ml

```diff
@@ -1373,18 +1371,26 @@ end = struct
 
   (* The current version of the rule digest scheme. We should increment it when
      making any changes to the scheme, to avoid collisions. *)
-  let rule_digest_version = 7
+  let rule_digest_version = 8
 
   let compute_rule_digest (rule : Rule.t) ~deps ~action ~sandbox_mode
       ~execution_parameters =
-    let { Action.Full.action; env; locks; can_go_in_shared_cache } = action in
+    let { Action.Full.action
+        ; env
+        ; locks
+        ; can_go_in_shared_cache
+        ; sandbox = _ (* already taken into account in [sandbox_mode] *)
```

Comment:
This seems a bit fishy: perhaps, we should no longer pass `sandbox_mode` to `compute_rule_digest`?

Reply:
@jeremiedimino convinced me that this is OK offline.